### ISSUE TITLE
Skype4pidgin does not automatically launch Skype Portable

### ISSUE THREAD

**Issue by GoogleCodeExporter** [EionRobb/skype4pidgin#90]
```
What steps will reproduce the problem?
1. Install Skype Portable and Pidgin Portable with libskype.dll and setup 
accounts.
2. Launch Skype Portable, then Pidgin Portable, confirm Skype to allow 
Pidgin access and go to silent mode. Skype contacts appear in Pidgin.
3. Close all apps. Relaunch Pidgin Portable, Skype does not launch, Pidgin 
reports Skype disconnected. Likewise if Skype is running but in offline 
mode.

What is the expected output? What do you see instead?
With an ordinary installed version of Skype, Pidgin Portable can 
automatically launch Skype. With the new Skype Portable this doesn't 
happen.

What version of the plugin are you using? On what operating system?
? The version dated 20 September 2009. Windows Vista.

Please provide any additional information below.

In summary, so long as Skype is already running and online before Pidgin 
(or the Skype account in Pidgin) launches the plugin loads the contacts and 
puts Skype into silent mode. However, if Skype is not running it simply 
reports Skype is offline and fails.

The version of Skype is 4.1.0.179. I cant confirm which build of Skype 
Portable I am using because the PortableApps site is down today. By the way 
this is the recent PortableApps Skype Portable and not any other version 
from more dubious sources.

Also, I am running these from a partitioned local drive rather than 
 a portable drive or USB stick.
```

Original issue reported on code.google.com by `panagopu...@gmail.com` on 3 Nov 2009 at 10:52
